### ISSUE TITLE
List products for an individual label

### ISSUE THREAD

**Issue by kozlown** [openfoodfacts/openfoodfacts-nodejs#11]
Create method `getProductsByLabel(label)`.

It must work for the following endpoint:
- https://en.wiki.openfoodfacts.org/API#Individual_label

**Comment by akashshetty1997** [comment]
i would love too work on it.

**Comment by TheMineWay** [comment]
The documentation seems to be outdated, as it does't show endpoint details. Where can we found the endpoint details?

**Comment by TheMineWay** [comment]
The endpoint documentation URL seems to be outdated. Is [this](https://[openfoodfacts.github.io/openfoodfacts-server/api/ref-v2/](https://openfoodfacts.github.io/openfoodfacts-server/api/ref-v2/#get-/api/v2/search)#get-/api/v2/search) the one we need to use?
I read we can use the "labels_tags" to filter by tags.
<img width="864" height="101" alt="Image" src="https://github.com/user-attachments/assets/c42b069e-9179-42cc-b1a3-48dc1e262792" />

Sounds logical?

**Comment by alexgarel** [comment]
@TheMineWay yes it's definitely a good idea to use the api v2 for this job !

**Comment by TheMineWay** [comment]
I have just released a PR for this. Finally I used API v3 :)
